src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
   - changeSet:
       id: create-books-table
-      author: nikita
+      author: team

```

Comment:
I mean that by author field your colleagues should identify the author of changes. As far as I know you name is not team, so please make author Mykyta Zuber